src/workos.ts

```diff
@@ -115,6 +118,10 @@ export class WorkOS {
     return new Webhooks(new SubtleCryptoProvider());
   }
 
+  createActionsClient() {
+    return new Actions(new SubtleCryptoProvider());
```

Comment:
Last piece of the puzzle (and I know we're inconsistent here), you should [return an error here](https://github.com/workos/workos-node/blob/main/src/workos.ts#L130-L132) and implement `createActionsClient` in `index.ts` as well as `index.worker.ts`.

Reply:
Sounds like there was an async convo, and we're not going to make this change.